tests/test_pyopener.py

```diff
@@ -13,17 +13,6 @@
 from rasterio.errors import OpenerRegistrationError
 
 
-def test_registration_failure():
```

Comment:
Do you think that a modified version of this test that succeeds would be useful?